### DISCUSSION TITLE
Student educational benefits (basic plan)

### DISCUSSION THREAD

**Discussion by jtmaveryk** [Labs64/NetLicensing-Community#108] [Questions] [answered]
**Question**
I remember seeing NetLicensing’s basic plan as a prior benefit of the GitHub Student Developer Pack and was curious if it was still available to get?

**Comment by r-brown** [comment, marked as answer]
Hello @jtmaveryk - please see  _Sunset of GitHub Student Developer Pack and Teacher Toolbox #50_ announcement.
Please send us the needed data as stated in the _"How To Apply"_ section and we'll enrol your vendor account after the verification.
